.sync.yml

```diff
@@ -3,3 +3,4 @@ Rakefile:
   param_docs_pattern:
     - manifests/init.pp
     - manifests/plugin/*.pp
+    - manifests/plugin/dns/*.pp
```

Comment:
This'll need a rebase after 39af736, but the fix in that commit will probably suffice for you too.